test/fixtures/minimal/variables.tf

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable project_name {
+  description = "The project to create test resources within."
+}
+
+variable "region" {}
```

Comment:
Variables should include descriptions.

Reply:
No problem. I didn't know if we considered descriptions important in examples/fixtures but I can see the case. Added.